catroid/src/main/java/org/catrobat/catroid/utils/CrashReporter.java

```diff
@@ -35,16 +34,16 @@
 import com.google.gson.Gson;
 
 import org.catrobat.catroid.BuildConfig;
-import org.catrobat.catroid.ui.MainMenuActivity;
 import org.catrobat.catroid.ui.SettingsActivity;
 
 import io.fabric.sdk.android.Fabric;
 
+import static org.catrobat.catroid.ui.BaseExceptionHandler.RECOVERED_FROM_CRASH;
+
```

Comment:
Code related to crash recovering should me moved to other classes/files.

I can't comment on line `preferences.edit().putBoolean(RECOVERED_FROM_CRASH, true).commit();`. This is the only line depending on RECOVERED_FROM_CRASH. When you move this line to the BaseExceptionHandler, everything related to the crash recovering is removed from the CrashReporter :man_dancing: